github-actions/create-pr-for-changes/lib/main.ts

```diff
@@ -129,6 +129,7 @@ async function main(): Promise<void> {
     const {
```

Comment:
Note: We could simplify this logic, if we keep re-using the same branch (like renovate)